tag-from-pipeline.sh

```diff
@@ -10,7 +10,7 @@ function get_version_in_changelog() {
     version_line=$(sed "${i}q;d" CHANGELOG.rst) # Get ith line of file
     set -- $version_line
     version=$2
-    if [[ $version =~ ^[0-9]+\.[0-9]+$ ]]; then
+    if [[ $version =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
`printf "\033[0;31mChangelog file is incorrect, one of the first seven lines should be of the format 'Version xx.xx.'\033[0m";` on the line 20 should be updated to `printf "\033[0;31mChangelog file is incorrect, one of the first seven lines should be of the format 'Version xx.xx.xx.'\033[0m";`

Reply:
Fixed